### DISCUSSION TITLE
Retrieve input file name (with path) in Lua filter

### DISCUSSION THREAD

**Discussion by unitscan** [jgm/pandoc#9860] [General]
If I launch this command

`pandoc --from latex+raw_tex --lua-filter=filter.lua -s test.tex -o test.docx`

there's a way to save the input file comprensive of his path (e.g. "C:\Logs\test.tex") in a variable?

**Comment by tarleb** [comment]
The list of input files is available via `PANDOC_STATE.input_files` (docs [1](https://pandoc.org/lua-filters#type-commonstate), [2](https://pandoc.org/lua-filters#global-variables)).

One can get an absolute path with `pandoc.path.join{pandoc.get_working_directory(), MY_RELATIVE_FILENAME}`.
